Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -92,6 +92,7 @@ try {
         Write-Host (Get-VstsLocString -Key SkippingIndexing)
     } else {
         Import-Module -Name $PSScriptRoot\IndexHelpers\IndexHelpers.psm1
+        $pdbFiles = $fileList | Where-Object { $_.EndsWith(".pdb", [StringComparison]::OrdinalIgnoreCase) }
```

Comment:
@arunkm Did you mean to rename $pdbFiles here too? I didn't see it referenced anymore below.

Reply:
No, Its used in line 96 (just after this line) (Invoke-IndexSources -SymbolsFilePaths $pdbFiles ...)

---
In reply to: [135646731](https://github.com/Microsoft/vsts-tasks/pull/5126#discussion_r135646731) [](ancestors = 135646731)